### PR TITLE
intital protobuf conversion for comment

### PR DESCRIPTION
**Pull request by saupchurch**
WIP for comment to see if the conversions steps are complete

### CODE REVIEW

File: src/main/resources/avro/rnaquantification.avdl [hunk 1]
```diff
@@ -1,140 +1,95 @@
...
+// number of reads in millions (FPM - Fragments Per Million).  FPM is then
+// divided by feature length in kilobases to obtain FPKM.
+
+// TPM - Transcripts per kilobase Per Million reads
+// TPM is calculated by first dividing the fragment/read count by feature length
+// in kilobases (RPK - Reads Per Kilobase).  The count of all RPKs in the sample
+// are then divided by a million to generate a 'per million' scaling value.  For
+// each feature RPK divided by the 'per million' scaling factor generated TPM.
+
 enum ExpressionUnits {
```
Comment (calbach) [inline src/main/resources/avro/rnaquantification.avdl]: nitpick: Enum names are typically singular.

File: src/main/resources/avro/rnaquantification.avdl [hunk 2]
```diff
@@ -1,140 +1,95 @@
...
-  FPKM,
-  TPM
+  FPKM = 0;
```
Comment (calbach) [inline src/main/resources/avro/rnaquantification.avdl]: Note: FPKM will be the default value if left uninitialized in an ExpressionLevel message. If this is not acceptable, then use something like EXPRESSION_UNIT_UNSPECIFIED as the default value. I use this pattern in a few places in my proto branch PR: https://github.com/ga4gh/schemas/pull/547

I don't have enough context to make a judgement here, but I'd guess you would want to use the unspecified enum value approach.

**Comment by saupchurch** [comment]
incorporated comment from @calbach and added conversion of methods

File: src/main/resources/avro/rnaquantificationmethods.avdl [hunk 3]
```diff
@@ -1,207 +1,146 @@
...
-  To get the next page of results, set this parameter to the value of
-  'nextPageToken' from the previous response.
-  */
-  union { null, string } nextPageToken = null;
+// This request maps to the URL `GET /rnaquantification/{rna_quantification_id}`.
+
+message GetRnaQuantificationRequest {
+  // The ID of the `RnaQuantification`.
+  string rna_quantification_id = 1;
+
```
Comment (calbach) [inline src/main/resources/avro/rnaquantificationmethods.avdl]: }

These issues should be caught by compilation via mvn once you've rebased onto the protobuf topic branch.

File: src/main/resources/avro/rnaquantificationmethods.avdl [hunk 4]
```diff
@@ -1,207 +1,146 @@
...
-SearchQuantificationGroupResponse searchQuantificationGroup(
-    /**
-    This request maps to the body of 'POST /quantificationgroup/search'
-    as JSON.
-    */
-    SearchQuantificationGroupRequest request) throws GAException;
+// This is the response from 'POST /quantificationgroup/search' expressed as JSON.
+message SearchQuantificationGroupResponse {
+  // The list of matching quantification groups.
+  repeated QuantificationGroup quantification_group = 1;
```
Comment (calbach) [inline src/main/resources/avro/rnaquantificationmethods.avdl]: Not directly related to proto, but these repeated response fields are plural in the other APIs, i.e. quantification_groups

**Comment by macieksmuga** [comment]
Should we just do the pluralizing here before we merge? ie. `SearchRnaQuantification` -> `SearchRnaQuantifications` etc...

**Comment by saupchurch** [comment]
I think we should do that - no sense making changes in schema that are just going to be superseded rapidly

**Comment by macieksmuga** [comment]
We'll need to rebase your changes against the main proto branch, which now has a different document hierarchy (`schemas/src/main/proto/ga4gh/*.proto`)

**Comment by macieksmuga** [comment]
It may just be cleanest to start a new PR with the revised folder organization.

**Comment by saupchurch** [comment]
files moved to correct location

**Pull request merged**